assets/js/program.js

```diff
@@ -0,0 +1,58 @@
+const programs = [{
+  icon: '<i class="fa fa-users fa-2x"></i>',
+  title: 'Network',
+  details: 'Network and interact with top industry individuals with many years of experience.',
+},
+
+{
+  icon: '<i class="fa fa-rocket fa-2x"></i>',
+  title: 'Workshops',
+  details: 'Software exhibitions, idea box, hackerton, code fest with so many prices to be won',
+},
+
+{
+  icon: '<i class="fa fa-comments-o fa-2x"></i>',
+  title: 'Forum',
+  details: 'Get opportunity to join forum of great minds, test your intellectual prowess',
+},
+
+{
+  icon: '<i class="fa fa-microphone fa-2x"></i>',
+  title: 'Talks',
+  details: 'Lets talk about security, software development models,  and so much more',
+},
+{
+  icon: '<i class="fa fa-graduation-cap fa-2x"></i>',
+  title: 'Certifications',
+  details: 'Certificate of participation will be given out to all those in attendance',
+},
+];
```

Comment:
- [ ] For readability purposes, I think we can format this array of objects better. 
We can do this:point_down:
```
const programs = [
   {
     ...
   },
   {
     ...
   },
     ...
 ];
 ```
 What do you think:thinking: